### PR TITLE
Math JS Examples

### PR DESCRIPTION
**Pull request by maboa**
Note (and please verify) the browser differences for the following examples:

Math.E

```
console.log((1 + (1/1000000)) ** 1000000);
// expected output: 2.7182804691564275 (Firefox/Safari)
// expected output: 2.7182804690957534 (Chrome)
```

Math.exp()

```
console.log(Math.exp(1));
// expected output: 2.718281828459045 (Chrome/Safari)
// expected output: 2.7182818284590455 (Firefox)
```

Math.atanh()
```
console.log(Math.atanh(0.5));
// expected output: 0.5493061443340548 (Firefox/Chrome)
// expected output: 0.5493061443340549 (Safari)
```

### CODE REVIEW

File: live-examples/js-examples/math-log1p.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<code id="static-js">console.log(Math.log10(100000));
+// expected output: 5
+
+console.log(Math.log10(2));
+// expected output: 0.3010299956639812
+
+console.log(Math.log10(1));
+// expected output: 0
+
+console.log(Math.log10(0));
```
Comment (wbamberg) [inline live-examples/js-examples/math-log1p.html]: This is the wrong function.

File: site.json [hunk 2]
```diff
@@ -1024,6 +1024,307 @@
...
+        "mathPi": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/math-pi.html",
+            "fileName": "math-pi.html",
+            "title": "JavaScript Demo: Math.PI",
+            "type": "js"
+        },
+        "mathPow": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/math-pi.html",
```
Comment (wbamberg) [inline site.json]: This is the wrong example

File: live-examples/js-examples/math-asinh.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<code id="static-js">console.log(Math.acosh(0.999999999999));
+// expected output: NaN
+
+console.log(Math.acosh(1));
+// expected output: 0
+
+console.log(Math.acosh(2));
+// expected output: 1.3169578969248166
+
+console.log(Math.acosh(2.5));
```
Comment (wbamberg) [inline live-examples/js-examples/math-asinh.html]: This is the wrong example

Comment (maboa) [reply live-examples/js-examples/math-asinh.html]: oops sorry - not sure how that slipped thru :/

File: live-examples/js-examples/math-cbrt.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+console.log(Math.cbrt(1));
+// expected output: 1
+
+console.log(Math.cbrt(Infinity));
+// expected output: Infinity
+
+console.log(Math.cbrt(2));
+// expected output: 1.2599210498948732
+</code>
```
[the remark this answers is not in the document]
Comment (maboa) [reply live-examples/js-examples/math-cbrt.html]: Yup.

File: live-examples/js-examples/math-log1p.html (hunk 1 above)
Comment (maboa) [reply live-examples/js-examples/math-log1p.html]: arghh

File: site.json (hunk 2 above)
Comment (maboa) [reply site.json]: Yup a site.json hiccup.

**Comment by maboa** [comment]
Where output varies across browser I have approximated the expected output and indicated that. Let me know if I can help with branch conflicts.

File: live-examples/js-examples/math-asinh.html (hunk 3 above)
Comment (wbamberg) [reply live-examples/js-examples/math-asinh.html]: I think this one still needs to be done, or am I confused?

**Comment by maboa** [comment]
Oops. Sorry about that - not sure how I missed. It should be there now @wbamberg

**Pull request merged**